src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ImageComponent.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+import { Grid, GridJustification, makeStyles } from '@material-ui/core';
+import { ITextResourceBindings } from 'src/features/form/layout';
+import { HelpTextContainer } from 'src/features/form/components/HelpTextContainer';
+import { IRuntimeState } from '../../types';
+
+export interface IImageProps {
+  id: string;
+  image: IImage;
+  textResourceBindings: ITextResourceBindings;
+  language: any;
+  getTextResource: (key: string) => string;
+  getTextResourceAsString: (key: string) => string;
+}
+
+export interface IImage {
+  src: IImagesrc;
+  width: string;
+  align: GridJustification;
+}
+
+export interface IImagesrc {
+  nb: string;
+  nn: string;
+  en: string;
+}
+
+const useStyles = makeStyles({
+  spacing: {
+    letterSpacing: '0.3px',
+  },
+});
+
+export function ImageComponent(props: IImageProps) {
+  const classes = useStyles();
+  const language: string =
+    useSelector((state: IRuntimeState) => state.profile.profile.profileSettingPreference.language);
+  const width = props.image.width || '100%';
+  const align = props.image.align || 'center';
+  const altText = props.getTextResourceAsString(props.textResourceBindings.altTextImg) || 'image';
+
+  let imgSrc;
```

Comment:
imgSrc could be a const and the tests down to line 50 could be a function with unit-test for picking image based on language.
Alternatively you can set the imageSrc directly as on line 49 and remove line 48 and 49.

Reply:
Like this?